src/Microsoft.Build.Tasks.Git/build/Microsoft.Build.Tasks.Git.props

```diff
@@ -2,6 +2,7 @@
 <Project>
   <PropertyGroup>
     <MicrosoftBuildTasksGitAssemblyFile Condition="'$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)..\tools\net472\Microsoft.Build.Tasks.Git.dll</MicrosoftBuildTasksGitAssemblyFile>
-    <MicrosoftBuildTasksGitAssemblyFile Condition="'$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tools\core\Microsoft.Build.Tasks.Git.dll</MicrosoftBuildTasksGitAssemblyFile>
+    <MicrosoftBuildTasksGitAssemblyFile Condition="'$(MSBuildRuntimeType)' == 'Core' and '$(MSBuildVersion)' &lt; '17.8.0'">$(MSBuildThisFileDirectory)..\tools\net6.0\Microsoft.Build.Tasks.Git.dll</MicrosoftBuildTasksGitAssemblyFile>
```

Comment:
Why not keep the directory name "core" so that we don't need to add (and maintain) these extra conditions?

Reply:
Grouping assemblies into a TFM subfolder is the pattern that we follow in other tools packages (i.e. Arcade) and aligns with NuGet's conventions for lib/ref folders.

Source-build requires to target the latest TFM and back-compatibility with older SDKs suggests to target the minimum supported tool TFM (net6.0 at the moment). To align with these goals, we recommend to build the super-set of TFMs in the regular build and make source build only build the latest TFM graph slice. 

Now that projects use floating TFM properties (`NetToolCurrent`, `NetToolMinimum` and `NetFrameworkCurrent`) the remaining places that need a manual TFM update (every year) are these hardcoded values in props and targets files.

I opened https://github.com/dotnet/arcade/issues/14138 to track adding infrastructure to Arcade to avoid these hardcodes via template substitution.

Reply:
> Grouping assemblies into a TFM subfolder is the pattern that we follow in other tools packages (i.e. Arcade) and aligns with NuGet's conventions for lib/ref folders.

I don't see the value of it for build tasks targets though since we are not multi-targeting them. We only ever need a one target for desktop and one for core, no?

Not using the TFMs would reduce complexity.